ironfish-cli/src/commands/miners/pools/status.ts

```diff
@@ -109,7 +110,8 @@ export class PoolStatus extends IronfishCommand {
 
     if (status.addressStatus) {
       result += `\nMining status for address '${status.addressStatus.publicAddress}':\n`
-      result += `Miners:                ${status.addressStatus.miners}\n`
+      result += `Number of miners:      ${status.addressStatus.miners}\n`
+      result += `Active miners:         ${status.addressStatus.activeMiners}\n`
```

Comment:
these are not necessarily active miners, just connected miners. I would expect an active miner to be someone with submitted shares. As @hughy said below, I assume you want this to be miners tied to your public key.